sdk/search/azure-search-documents/azure/search/documents/_index_documents_batch.py

```diff
@@ -111,23 +103,21 @@ def add_merge_or_upload_actions(
         :param documents: Documents to merge or upload into an Azure search
          index. May be a single list of documents, or documents as individual
          parameters.
-        :type documents: dict or list[dict]
+        :type documents: Dict or List[Dict]
         :return: the added actions
         :rtype: List[IndexAction]
         """
         return self._extend_batch(_flatten_args(documents), "mergeOrUpload")
 
     @property
-    def actions(self):
-        # type: () -> List[IndexAction]
+    def actions(self) -> List[IndexAction]:
         """The list of currently index actions to index.
 
         :rtype: List[IndexAction]
         """
         return list(self._actions)
 
-    def dequeue_actions(self, **kwargs):  # pylint: disable=unused-argument
-        # type: () -> List[IndexAction]
+    def dequeue_actions(self, **kwargs) -> List[IndexAction]:  # pylint: disable=unused-argument
```

Comment:
I think for kwargs we have settled on `Any` but if moving to py3 style kwargs those can be typed as well